src/core/thread/mle.cpp

```diff
@@ -2139,10 +2139,6 @@ ThreadError Mle::HandleChildIdResponse(const Message &aMessage, const Ip6::Messa
             mNetif.GetPendingDataset().Set(pendingTimestamp, aMessage, offset + sizeof(tlv), tlv.GetLength());
         }
     }
-    else
```

Comment:
If the Active Timestamp is 0, it means that the parent doesn't have valid Active Operational Dataset, so I add the check logic. Now, including a 0 value Active Timestamp TLV has the same meaning with the case that not including Active Timestamp TLV in previous implementation.